packages/runtime/src/runtime.ts

```diff
@@ -355,7 +357,7 @@ export class Runtime {
       amount: address === creatorAddr ? assetDef.total : 0, // for creator opt-in amount is total assets
       'asset-id': assetIndex,
       creator: creatorAddr,
-      'is-frozen': assetDef["default-frozen"]
+      'is-frozen': address === creatorAddr ? false : assetDef["default-frozen"]
```

Comment:
can you please elaborate why are we doing this ?

Reply:
Because for creator even if the asset is default frozen, for his holding the frozen is `false`. Confirmed by testing.
```
Note: A frozen account can always close out to the asset creator
```
Reference: [https://developer.algorand.org/tutorials/asa-javascript/](https://developer.algorand.org/tutorials/asa-javascript/)